src/Command/RemindEndingCfpCommand.php

```diff
@@ -23,14 +24,16 @@ class RemindEndingCfpCommand extends Command
 {
     const REMAINING_DAYS_STEPS = [30, 20, 10, 5, 1, 0];
 
-    private $repository;
+    private ConferenceRepository $repository;
     private $eventDispatcher;
 
     public function __construct(ManagerRegistry $doctrine, EventDispatcherInterface $eventDispatcher)
     {
         parent::__construct();
 
-        $this->repository = $doctrine->getManager()->getRepository(Conference::class);
+        /** @var ConferenceRepository $conferenceRepository */
+        $conferenceRepository = $doctrine->getManager()->getRepository(Conference::class);
```

Comment:
This is a bad practice to inject the Entity Manager to get ta Repository. Inject the repository instead.
It's a bad practice because
* It hides a deps
* class become harder to test
(same for Symfony Container)